payload-schemas/schemas/deployment_status/created.schema.json

```diff
@@ -35,7 +35,9 @@
         "creator": { "$ref": "common/user.schema.json" },
         "description": { "type": "string" },
         "environment": { "type": "string" },
-        "target_url": { "type": "string" },
+        "target_url": {
+          "oneOf": [{ "type": "string", "format": "uri" }, { "enum": [""] }]
```

Comment:
`deployment_status#target_url` can be "" https://github.com/octokit/webhooks/blob/46983cd8739c706639c066eaed538c768c4d9cc3/payload-examples/api.github.com/deployment_status/payload.json#L30